### PR TITLE
added about-us section

### LINKED ISSUE #55: Create About us Section
**Deliverables**
- Make Generalized components.
- Also Generalized color as well. (I mean we can pass using props).

**Sample**
Capture2
Capture6
Capture9

**Additional context**
- Tech/Framework:- React Js

### PR DESCRIPTION
**Pull request by shreyanshi16**
# Description

Added About Us, Newsletter, and Join Us section

Closes #55 

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [✓] My code follows the style guidelines of this project
- [✓] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [✓] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by shreyanshi16** [comment]
@codewithvk please check my PR.

**Comment by codewithvk** [comment]
hey @shreyanshi16 , Could you please attach a screenshot?

**Comment by shreyanshi16** [comment]
image
image

Hey here are the screenshots!

File: frontend/src/Components/Common/Common.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  return (
+    <Background id="header">
+      <Base>
+        <div className="container-fluid">
+          <div className="row">
+            <div className="col-10 mx-auto content">
+              <div className="row">
+                <div className=" col-sm-8 order-1 order-lg-1 d-flex justify-content-center flex-column">
+                  <Group>
+                    <Ellipse1></Ellipse1>
```
[the remark this answers is not in the document]
Comment (shreyanshi16) [reply frontend/src/Components/Common/Common.js]: Done

**Comment by codewithvk** [comment]
Thanks for contributing @shreyanshi16 !

**Pull request merged**